building/tracks/syllabus/README.md

```diff
@@ -0,0 +1,177 @@
+
+## Porting and borrowing
+
+It can be worth looking at how other language tracks have built out their concept exercises.
+You can find examples of Concept Exercises from other language tracks [here](https://exercism.org/docs/building/tracks/stories).
+
+That said, if you decide to use other exercises as a starting point for your own, be careful to ensure that the resulting exercise is about the concept as it exists in your language.
+Sometimes concepts differ subtly, sometimes radically.
+Sometimes concepts don't exist at all.
+
+The syllabus, and therefore the concept tree, should represent the concepts that exist in the language.
+
+Don't include concepts just because other tracks do.
+
+In some cases it might be tempting to put a concept in because people often have to work around it by using concepts that do exist.
+Rather than doing this, introduce the concept that the language _does_ use, and consider adding an exercise that explains how to use it in that type of situation.
+
+For example, in Go there are no enums.
+Instead the Go concept tree introduces constants, and teaches how to use constants in the type of situation where you might use enums in other languages.
+
+## Asking for help
+Don't hesitate to ask for help.
+It's better to ask up front or while working on an exercise rather than discussing during code review.
+
+On GitHub you can mention the team @exercism/learning-mode-consultants.
+In Slack you can ask in the #learning-mode channel.
+
+## Getting started
+
+Our experience has taught us that the most pragmatic way to develop a syllabus is to grow the concept tree organically, starting with the simplest concepts.
+We don't have to get everything right up front, but it's really useful to not think too far ahead.
+
+We start with the track's easiest concepts.
+
+Some things are easy because they are inherently simple.
+Other things are not all that simple, but feel easy because they are familiar.
+Familiar is good.
+Familiar is not confusing.
+
+While the endpoint is to write idiomatic code, the stepping stones to get there are not always idiomatic.
+Using what is familiar — even if it is not a great example of code in that language — helps move a student more quickly toward the goal of code that is more typical of the language.
+
+### Developing the first exercise
+
+Rather than trying to map out the entire concept tree up front, just start with the first exercise.
+Think of this exercise as the "Hello World" of concept exercises.
+The goal of the first exercise is to allow the student to start learning with the least amount of friction as possible.
+They are taking the very first step towards getting familiar with what code in this language looks like.
+They might write a small piece of code, or perhaps just make a couple of additions to a stub in order to complete the exercise.
+Try to optimize for a quick win and getting students familiar enough with the bare necessities of syntax to be able to move forward confidently.
```

Comment:
Is it worth noting that a student does actually do "Hello World" before getting to the first concept exercise?